### PR TITLE
cork: drop deprecated enter method

### PR DESCRIPTION
**Pull request by ajeddeloh**
depends on https://github.com/coreos/scripts/pull/820

### CODE REVIEW

**Comment by ajeddeloh** [comment]
Passed jenkins

File: cmd/cork/create.go [hunk 1]
```diff
@@ -127,9 +126,7 @@ func init() {
 	enterCmd.Flags().AddFlagSet(chrootFlags)
 	enterCmd.Flags().BoolVar(&experimental,
 		"experimental", false, "Use new enter implementation")
-	enterCmd.Flags().MarkDeprecated("experimental", "--experimental is now default and the flag will be removed soon. Implies --bind-gpg-agent=false")
-	enterCmd.Flags().BoolVar(&legacy,
-		"legacy", false, "Use the old enter implementation")
+	enterCmd.Flags().MarkDeprecated("experimental", "--experimental is deprecated and will be removed next release. Use --bind-gpg-agent=false")
```
Comment (bgilbert) [inline cmd/cork/create.go]: `Use` is probably too strong; it's true for Jenkins but probably not for anyone else.

Comment (dm0-) [reply cmd/cork/create.go]: Should the flag be changed in the scripts PR as part of this change set?  The change isn't required, but if we're removing it in the next release, we might as well update.

Comment (ajeddeloh) [reply cmd/cork/create.go]: @bgilbert how about `This is equivilent to --bind-gpg-agent=false`?

Comment (bgilbert) [reply cmd/cork/create.go]: Could just be `Equivalent to --bind-gpg-agent=false`, but sure.  I'm also okay with the wording of the existing second sentence.

**Pull request merged**